docs/statistics/probability/LawOfLargeNumbers.md

```diff
@@ -62,9 +63,7 @@
 
 
     
-## Sample Estimate Distribution
-## Sampling Varibility
-## Law of Large Numbers 
+xxxxxxxxxxxxxxx
```

Comment:
I guess that's just a placeholder?

Reply:
yeah. this chapter is work in progress :)